certified-connectors/JG Integrations/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@jglassonnz Please remove icon.png and settings.png files from this PR, these two files are not required to ope-source.